-versions.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  required_providers {
+    aws = {
+      version = ">= 3.63"
```

Comment:
I think AWS provider 3.28 was when conformance packs were added: https://github.com/hashicorp/terraform-provider-aws/releases?q=config_conformance_pack&expanded=true

Reply:
I see that the new version 3.71. Could you elaborate on the concern? @rchildress87